mmpose/core/post_processing/post_transforms.py

```diff
@@ -62,26 +62,34 @@ def flip_back(output_flipped, flip_pairs):
             from the flipped images.
         flip_pairs (list[tuple()): Pairs of keypoints which are mirrored
             (for example, left ear -- right ear).
+        target_type (str): GaussianHeatMap or CombinedTarget
 
     Returns:
         np.ndarray: heatmaps that flipped back to the original image
     """
     assert output_flipped.ndim == 4, \
         'output_flipped should be [batch_size, num_keypoints, height, width]'
 
```

Comment:
add `assert target_type in ('GaussianHeatMap', 'CombinedTarget')`

Reply:
done